From be3f218b6964116bc2948a9b7b93247322dc979f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ale=C5=A1=20Mat=C4=9Bj?= <amatej@redhat.com>
Date: Thu, 12 Dec 2024 12:59:03 +0100
Subject: [PATCH] doc: `--disableexcludepkgs=all` doesn't affect just file
 configuration

The option `--disableexcludepkgs=all` disables all configuration
includes and excludes including packages specified on the commandline
via `--exclude=`, `-x=`, the deprecated `--excludepkgs=` and file
configuration via `--excludepkgs=`, `--includepkgs=`.
---
 doc/command_ref.rst | 9 +++++----
 1 file changed, 5 insertions(+), 4 deletions(-)

diff --git a/doc/command_ref.rst b/doc/command_ref.rst
index 36817c00..75ded68c 100644
--- a/doc/command_ref.rst
+++ b/doc/command_ref.rst
@@ -165,12 +165,13 @@ Options
 .. _disableexcludes-label:
 
 ``--disableexcludes=[all|main|<repoid>], --disableexcludepkgs=[all|main|<repoid>]``
+    Disable ``excludepkgs`` and ``includepkgs`` configuration options. Takes one of the following three options:
 
-    Disable the configuration file excludes. Takes one of the following three options:
+    * ``all``, disables all ``excludepkgs`` and ``includepkgs`` configurations
+    * ``main``, disables ``excludepkgs`` and ``includepkgs`` defined in the ``[main]`` section
+    * ``repoid``, disables ``excludepkgs`` and ``includepkgs`` defined for the given repository
 
-    * ``all``, disables all configuration file excludes
-    * ``main``, disables excludes defined in the ``[main]`` section
-    * ``repoid``, disables excludes defined for the given repository
+    Note that the \-\ :ref:`-exclude <exclude_option-label>` option appends to the ``[main]`` ``excludepkgs`` configuration and therefore is disabled when ``main`` or ``all`` is specified.
 
 ``--disable, --set-disabled``
     Disable specified repositories (automatically saves). The option has to be used together with the
-- 
2.46.2

